examples/web-wagmi/src/utils/media.ts

```diff
@@ -0,0 +1,65 @@
+import { ContentPublication, Overwrite } from '@lens-protocol/react-web';
+
+import { resolveFullResourceUrl } from './resolveFullResourceUrl';
+
+export const isInEnum = <TEnumValue extends string | number>(
+  enumVariables: { [key: string]: TEnumValue },
+  value: string | number,
+): value is TEnumValue => {
+  return Object.values<string | number>(enumVariables).includes(value);
+};
+
+export enum ImageType {
+  JPEG = 'image/jpeg',
+  JPG = 'image/jpg',
+  PNG = 'image/png',
+  WEBP = 'image/webp',
+  GIF = 'image/gif',
+}
+
+export enum VideoType {
+  MP4 = 'video/mp4',
+}
+
+export enum AudioType {
+  WAV = 'audio/wav',
+  MP3 = 'audio/mpeg',
+  OGG = 'audio/ogg',
+}
+
+export const FileType = { ...ImageType, ...VideoType, ...AudioType };
+export type FileType = ImageType | VideoType | AudioType;
+
+type Media = ContentPublication['metadata']['media'][number]['original'];
```

Comment:
We should probably export `Media`

Reply:
it is exported, I didn't check, I copied it straight from our other project, now it is fixed.